src/felles-komponenter/utils/logging.js

```diff
@@ -44,6 +44,31 @@ export const TILSTAND_FILTER_METRIKK = `${filterBase}Tilstand`;
 
 const AKTIVITET_FLYTTET = 'aktivitetsplan.aktivitet.flyttet';
 
+function hash(string) {
+    return string
+        ? shajs('sha256')
+              .update(string)
+              .digest('hex')
+        : null;
+}
+
+export function loggAntalVeiledere(servicegruppe, underOppfolging, ident, aktorId) {
+    const fields = {
+        underOppfolging,
+        veileder: hash(ident),
+        bruker: hash(aktorId),
+    };
+    loggEvent(ANTALL_VEILEDERE, fields, servicegruppe);
+}
+
+export function loggingAntallBrukere(servicegruppe, underOppfolging, aktorId) {
+    if (!underOppfolging) {
+        loggEvent(LOGG_BRUKER_IKKE_OPPFOLGING, underOppfolging, servicegruppe);
+    } else {
+        loggEvent(LOGGING_ANTALLBRUKERE, hash(aktorId), servicegruppe);
```

Comment:
Er det meningen at bare de som er under oppfølging skal ha med hashet aktørid? Forstår ikke helt hva denne loggingen er for og endringene er ikke koblet opp mot noen brukerhistorie

Reply:
kan ikke lagre Peron informasjon om brukere som ikke er under oppfølging